package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vl-ui-modal",
-  "version": "3.0.5",
+  "version": "3.0.4",
```

Comment:
Is dit ok?

Reply:
Nee, blijkbaar niet up to date met de master. Ik heb het aangepast.